lib/services/page-service.ts

```diff
@@ -71,6 +85,93 @@ export class PageService implements IPageService {
                 });
         });
     }
+
+    private createPage(srcDir: string, destDir: string, pageName: string): Promise<string> {
+        const camelCaseNameString = _.camelCase(pageName);
+        const pageDirPath = util.path.join(destDir, pageName);
+
+        const newPageName = {
+            originalName: pageName,
+            pascalCaseName: _.upperFirst(camelCaseNameString),
+            camelCaseName: camelCaseNameString
+        };
+
+        return new Promise((resolve, reject) => {
+            this.prepareRenderedFiles(srcDir, newPageName)
+                .then((renderedPageFiles) => {
+                    util.fs.ensureDir(pageDirPath)
```

Comment:
Does this return a promise? If yes, you can simply use 'return' and continue the promise chain. If not, you should pass a callback instead of using then()